main.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from huggingface_pipelines.pipeline_config import TextPipelineConfig, MetricConfig
+from huggingface_pipelines.pipeline_factory import PipelineFactory
+from huggingface_pipelines.metric_analyzer_factory import MetricAnalyzerFactory
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def main():
+
+    pipeline_config = TextPipelineConfig(
```

Comment:
in call of `TextPipelineConfig` you can call different steps one after another (chaining).